idunn/directions/models.py

```diff
@@ -0,0 +1,196 @@
+from enum import Enum
+from typing import List, Optional, Tuple
+from pydantic import BaseModel, Schema, validator
+
+
+class TransportMode(str, Enum):
+    walk = 'WALK'
+    bicycle = 'BICYCLE'
+    car = 'CAR'
+    boat = 'BOAT'
+    plane = 'PLANE'
+    train = 'TRAIN'
+    carpool = 'CARPOOL'
+    bus = 'BUS'
+    bus_city = 'BUS_CITY'
+    vtc = 'VTC'
+    taxi = 'TAXI'
+    bike = 'BIKE'
+    tram = 'TRAM'
+    car_rental = 'CAR_RENTAL'
+    transfert = 'TRANSFERT'
+    subway = 'SUBWAY'
+    suburban_train = 'SUBURBAN_TRAIN'
+    seaplane = 'SEAPLANE'
+    helicopter = 'HELICOPTER'
+    funicular = 'FUNICULAR'
+    shuttle = 'SHUTTLE'
+    unknown = 'UNKNOW'
```

Comment:
The final 'N' is missing, I guess that's a typo.

Reply:
That's actually how it is documented on https://maps2.combigo.com/v1/api-docs/#/ (model `TransportType`). Maybe a typo on their side though.

I've just added a comment.